README.md

```diff
@@ -1,6 +1,6 @@
 # CSS Battle Challenge Solutions
 
-![Logo](./assets/logo.png)
+![Logo](https://abhijeetsaxena.in/css-battle-solutions/assets/logo.png)
```

Comment:
@rahulps1000 What is the purpose of this request ? Relative urls are better than hardcoding links with domain. What if the domain name changes or is no longer valid ?